manage.py

```diff
@@ -118,6 +97,86 @@ def analyze_all(self):
         )
         print('Batcher invocation successful!')
 
+    @staticmethod
+    def build():
+        """Build Lambda packages (saves *.zip files in terraform/)."""
+        lambda_build(TERRAFORM_DIR)
+
+    def deploy(self):
+        """Deploy BinaryAlert. Equivalent to test + build + apply + analyze_all."""
+        self.test()
+        self.build()
+        self.apply()
+        self.analyze_all()
+
+    def live_test(self):
+        """Upload an EICAR test file to BinaryAlert which should trigger a YARA match alert."""
+        bucket_name = '{}.binaryalert-binaries.{}'.format(
+            self._config['name_prefix'].replace('_', '.'), self._config['aws_region'])
+        test_filename = 'eicar_test_{}.txt'.format(uuid.uuid4())
+        s3_identifier = 'S3:{}:{}'.format(bucket_name, test_filename)
+
+        print('Uploading EICAR test file {}...'.format(s3_identifier))
+        bucket = boto3.resource('s3').Bucket(bucket_name)
+        bucket.put_object(
+            Body=EICAR_STRING.encode('UTF-8'),
+            Key=test_filename,
+            Metadata={'observed_path': test_filename}
+        )
+
+        table_name = '{}_binaryalert_matches'.format(self._config['name_prefix'])
+        print('EICAR test file uploaded! Connecting to table DynamoDB:{}'.format(table_name))
+        table = boto3.resource('dynamodb').Table(table_name)
+        eicar_sha256 = hashlib.sha256(EICAR_STRING.encode('UTF-8')).hexdigest()
+        dynamo_record_found = False
+        lambda_version = 0
+
+        for attempt in range(1, 11):
+            time.sleep(5)
```

Comment:
We don't strictly have to, but I wanted to have a little bit of wait instead of constantly hitting DyanmoDB with a query. You pay for each query, and it's possible to throttle the DB with too many requests. If the live test takes 10 seconds, why query 10 times when you need only query twice?